package.json

```diff
@@ -54,12 +55,6 @@
     "inferno-server": "^3.1.0",
     "isomorphic-fetch": "^2.2.1",
     "moment": "^2.18.1",
-    "@types/express": "^4.0.35",
-    "core-js": "^2.4.1",
-    "express": "^4.15.2",
-    "http-proxy-middleware": "^0.17.4",
-    "isomorphic-fetch": "^2.2.1",
-    "moment": "^2.18.1",
```

Comment:
are you sure we don't need these? try installing with [yarn](https://code.facebook.com/posts/1840075619545360) to make sure the package.json actually corresponds to your locally installed packages